src/coprocessor/dag/batch/executors/slow_hash_aggr_executor.rs

```diff
@@ -1,20 +1,22 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
+use std::hash::{Hash, Hasher};
+use std::ptr::NonNull;
 use std::sync::Arc;
 
 use hashbrown::hash_map::Entry;
-use smallvec::SmallVec;
 
```

Comment:
Remove this empty line (not a strong suggestion).

Reply:
TiKV doesn't have a specification on the format of these `use` now...I hope there will be one.